coupon_incompatibility/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+Once you've configured your incompatibilities, go to a quotations that fulfills your
```

Comment:
You should split the readme to fit each part purpose

Reply:
I had already made the split, but I forgot to remove this part.